backend/constants.js

```diff
@@ -0,0 +1,7 @@
+export const initialData = {
```

Comment:
Do we really need to repeat the initial data in `constants.js`? How about if someone deletes the entry what is returned an object that looks like `{ html: '', css: '', js: '' }`?

Reply:
maybe I'm doing something wrong, but the initial data template did not work for me. both of the files are in the project - any ideas?